src/storefront/src/templates/views/productCard.pug

```diff
@@ -66,11 +66,20 @@ div.tm-product-card-body
           
           +icon("heart", ".75")
           span.tm-product-card-action-text Add to favorites
+      
+      div.uk-text-meta.tm-product-card-actions
+          a.tm-product-card-action.tm-action-button-active(
+          title="Ratings")          
+          +icon("star")
+          +icon("star")
+          +icon("star")
+          +icon("star")
+          +icon("star")
```

Comment:
**NOTICE** to reviewers: this is an intentional failure

Reply:
I'll have to take this out, see previous comment on including this part of beta release. Keeping things simple.